src/material-experimental/mdc-list/list-base.ts

```diff
@@ -34,6 +34,54 @@ function toggleClass(el: Element, className: string, on: boolean) {
   }
 }
 
+class ListAdapter implements MDCListAdapter {
+  constructor(private _delegate: MatInteractiveListBase) {}
+
+  getListItemCount = () => this._delegate._items.length;
```

Comment:
can you put newlines between these and use class method syntax?

Reply:
Okay, will change that.